<launch>
  <arg name="input_topic_name" default="/move_base_simple/goal"/>
  <arg name="output_topic_name" default="/planning/mission_planning/route"/>
  <arg name="visualization_topic_name" default="/planning/mission_planning/route_marker"/>
  <node pkg="mission_planner" type="mission_planner" name="mission_planner" output="screen">
    <param name="map_frame" value="map"/>
    <param name="base_link_frame" value="base_link"/>
    <remap from="goal_pose" to="$(arg input_topic_name)"/>
    <remap from="route" to="$(arg output_topic_name)"/>
    <remap from="route_marker" to="$(arg visualization_topic_name)"/>
  </node>
</launch>
